packages/@lwc/integration-karma/scripts/karma-configs/utils.js

```diff
@@ -45,13 +42,8 @@ function getSauceSection({ suiteName, tags, customData }) {
 
         customData: {
             ...customData,
-
             ci: IS_CI,
-
-            build: CIRCLE_BUILD_NUM,
-            commit: CIRCLE_SHA1,
-            branch: CIRCLE_BRANCH,
-            buildUrl: CIRCLE_BUILD_URL,
+            buildId,
```

Comment:
I don't see the point of all this extra customData. I'm sure nobody ever looks at it anyway.